debian/control

```diff
@@ -2,8 +2,7 @@ Source: cloudstack
 Section: libs
 Priority: extra
 Maintainer: Wido den Hollander <wido@widodh.nl>
-Build-Depends: debhelper (>= 9), openjdk-11-jdk | java11-sdk | java11-jdk | zulu-11, genisoimage,
- python-mysql.connector | python3-mysql.connector, maven (>= 3) | maven3, python (>= 2.7), python3 (>= 3), nodejs (>= 12), lsb-release, dh-systemd, python-setuptools
```

Comment:
Removing some of the dependencies such as maven3, nodejs is not correct - these are build dependencies.

Reply:
Hello @rhtyd I disagree with your statement. My change doesn't affect the code base - only the build flow, which is currently over constrained. 

Over constrained means that you forced it to use deps versions, shipped with the distribution (Ubuntu 20.04 apt in this case).

Some packages are no longer available in 21.04, as mentioned #5350 . For some deps, like Maven and Node you don't get the latest version easily, which I easily do [here](https://github.com/tetra12/cloudstack-deb-builder/blob/7be9703ee9d031743c32be4c97ac124616593593/ubuntu2004/Dockerfile#L57).

I was able to build ACS and make it up&running with this script for Ubuntu 21.04 and a customized `qemu` and `libvirt`, cloned from git from master. Thus I guarantee that this works fine for Ubuntu 20.04+

If you don't see a value in supporting the latest versions of Ubuntu, Maven, Node, Qemu and Libvirt, feel free to close this PR

Reply:
Your change breaks packaging